vars/CentOS.yml

```diff
@@ -30,6 +30,7 @@ packages:
   - xz-devel
   - zchunk-devel
   - zlib-devel
+  - swig
```

Comment:
Will swig-2.0.10-5.el7 from CentOS 7 itself suffice, or must we use swig3-3.0.12-17.el7 from CentOS 7 extras?

If the latter, RHEL7 Extras repo would need to be enabled. (We already enabled RHEL7 optional.)

Reply:
And note that Fedora 30 "swig" is 3.0, while Fedora 31 has 4.0 .